mandatory/3-function-output.js

```diff
@@ -1,16 +1,15 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// Return a random number between 0 and 10
```

Comment:
What kind of number? Decimal? Integer? Positive or negative?

Reply:
Not answered.